### ISSUE TITLE
Lab 6 Null Hypothesis

### ISSUE THREAD

**Issue by dholford** [Watts-College/cpp-524-fall-2021#39]
I keep getting myself turned around on what the null hypothesis would be for TFA vs. Regular teachers. My sense is the null hypothesis would be that there is no difference and the alternate hypothesis is that there is a difference. 

Can someone confirm or set me straight?

**Comment by lecy** [comment]
That’s correct. With one group that has two levels, one level is included as a dummy, one excluded, the implicit test is the reference group (omitted category) versus the dummy variable group.

For example, if male omitted then b0 becomes the mean of male and b0+b1 is the mean of female.

Y(male) = Y(female) is the Null

which is operationalized as:

b0 = b0+b1  —>
0 = b1

The p-value tells us the likelihood the null is correct, so low p-value means b1 ≠ 0, which is the same as saying the group means Y(m) and Y(f) are NOT the same.
